setup.cfg

```diff
@@ -0,0 +1,47 @@
+[coverage:report]
+precision = 2
+exclude_lines =
+    @abc.abstractmethod
+    @abc.abstractproperty
+    NotImplementedError
+    pragma: no cover
+    __repr__
+    __str__
+
+[coverage:run]
+branch = True
+source =
+    src
+    tests
+omit =
+    tests/utils.py
```

Comment:
No need to omit anything unless there is a reason.

Reply:
That was already there :)

I'll remove this line.